app/models/manager_refresh/inventory_object_lazy.rb

```diff
@@ -36,7 +36,16 @@ def transitive_dependency?
       # and a :stack is a relation to another object, in the InventoryObject object,
       # then this relation is considered transitive.
       !!(key && (inventory_collection.dependency_attributes.keys.include?(key) ||
-        inventory_collection.model_class.reflect_on_association(key)))
+        association?(key)))
+    end
+
+    def association?(key)
+      # TODO(lsmola) remove this if there will be better dependency scan, probably with transitive dependencies filled
+      # in a second pass
+      return true if [:parent, :genelogy_parent].include?(key)
```

Comment:
typo — s/genelogy/genealogy/
Good luck figuring out the consequences :)

Reply:
hehe, we solve those dependencies manually so hopefully no consequence. :-)